pyne/alara.py

```diff
@@ -517,3 +519,149 @@ def irradiation_blocks(material_lib, element_lib, data_library, cooling,
     s += "dump_file {0}\n".format(dump_file)
 
     return s
+
+""" Chebyshev Rational Approximation Method functions
+"""
+
+def _build_matrix(N):
+    """ This function  builds burnup matrix, A. Decay only.
+    """
+    
+    A = np.zeros((len(N), len(N)))
+    
+    # convert N to id form
+    N_id = []
+    for i in xrange(len(N)):
+        ID = id(N[i])
+        N_id.append(ID)
+        
+    sds = SimpleDataSource()
+
+    # Decay
+    for i in xrange(len(N)):
+        A[i, i] -= decay_const(N_id[i])
+        
+        # Find decay parents
+        for k in xrange(len(N)):
+            if N_id[i] in decay_children(N_id[k]):
+                A[k, i] += decay_const(N_id[k])
+            
+    return A
+
+def _rat_apprx_14(A, t, n_0):
+    """ CRAM of order 14
+
+    Parameters
+    ---------
+
+    A : numpy array
+	Burnup matrix
+    t : float
+	Time step
+    n_0: numpy array
+	Inital composition vector
+    """
+	
+    theta = np.array([ complex(-8.8977731864688888199, 16.630982619902085304), 
+                   complex(-3.7032750494234480603, 13.656371871483268171), 
```

Comment:
It is probably better to use complex literals, eg

```python
-3.7032750494234480603 + 13.656371871483268171j
```
Notice the `j` at the end.  Silly electrical engineers.